pkg/sfu/forwarder.go

```diff
@@ -1058,27 +1064,49 @@ func (f *Forwarder) GetTranslationParams(extPkt *buffer.ExtPacket, layer int32)
 }
 
 // should be called with lock held
-func (f *Forwarder) getTranslationParamsAudio(extPkt *buffer.ExtPacket) (*TranslationParams, error) {
+func (f *Forwarder) getTranslationParamsCommon(extPkt *buffer.ExtPacket) (*TranslationParams, error) {
```

Comment:
Called this `Common` although there are a couple of bits about `vp8Munger` in here. Splitting it out would have made things uncouth. So, not doing it.